From 9aac4462f11504dec3b022fb0982967903e91e99 Mon Sep 17 00:00:00 2001
From: Kent Tamura <tkent@chromium.org>
Date: Mon, 6 Feb 2023 06:19:14 +0000
Subject: [PATCH] DeferredShaping: Disable the feature in all tests

Bug: 1259085
Change-Id: I6d00438b9dd44305b670125915dd2e07077f67d8
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4220915
Commit-Queue: Rakina Zata Amni <rakina@chromium.org>
Reviewed-by: Rakina Zata Amni <rakina@chromium.org>
Auto-Submit: Kent Tamura <tkent@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1101458}
---
 .../accessibility/render_accessibility_impl_browsertest.cc    | 4 ----
 .../blink/renderer/platform/runtime_enabled_features.json5    | 2 --
 2 files changed, 6 deletions(-)

--- a/content/renderer/accessibility/render_accessibility_impl_browsertest.cc
+++ b/content/renderer/accessibility/render_accessibility_impl_browsertest.cc
@@ -300,10 +300,6 @@
     blink::WebRuntimeFeatures::EnableAccessibilityExposeHTMLElement(true);
 
     RenderViewTest::SetUp();
-    // TestBoundsFor*FixedNodeAfterScroll doesn't work with DeferredShaping,
-    // which triggers layout of shaping-deferred elements on scrollTo().
-    blink::WebRuntimeFeatures::EnableFeatureFromString("DeferredShaping",
-                                                       false);
 
     sink_ = &render_thread_->sink();
 
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -983,8 +983,6 @@
     {
       // crbug.com/1259085
       name: "DeferredShaping",
-      status: "test",
-      base_feature: "DeferredShaping",
     },
     {
       name: "DeflateRawCompressionFormat",
